modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc

```diff
@@ -0,0 +1,115 @@
+= Anypoint MQ Connector - 3.x
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: Select
+
+The Anypoint MQ connector provides publish-subscribe messaging to Mule apps using the Anypoint MQ cloud service.
+
+Anypoint MQ supports a wide range of messaging use cases: enterprise messaging patterns, asynchronous communications between microservices, IoT, and more. Anypoint MQ also has a REST API for publishing or consuming messages using any framework such as node.js, Java, Go, and devops bash scripts.
+
+For more information, see the xref:mq::index.adoc[Anypoint MQ documentation set] and additional resources listed in <<See Also>>.
+
+== Prerequisites
+
+To use this connector, you should be familiar with Anypoint Studio, Anypoint connectors, and Mule flows. The Anypoint MQ connector is free, but connecting to the service in Anypoint Platform requires a license.
+
+=== Compatibility
+
+[%header%autowidth.spread]
+|===
+|Software |Version
+|Mule Runtime |4.1.1 and later
+|Anypoint Studio |v7 and later
+|===
+
+IMPORTANT: This documentation applies to Mule 4 only. For Mule 3 compatible connector information, see xref:3.9@mule-runtime::anypoint-mq-connector.adoc[Mule 3 MQ Connector].
```

Comment:
Same as in 2.x -- this note is not important and the link should just be in See Also